11ty/definitions/blacklist-whitelist.md

```diff
@@ -0,0 +1,34 @@
+---
+title: Blacklist Whitelist
+slug: blacklist-whitelist
+flag:
+  level: avoid
+  text: racist term
+defined: true
+exceprt: terms used in many contexts to represent a list of entities or people who are to be allowed or denied access to a service, privilege, or recognition. To "blacklist" is "to exclude", to "whitelist" means "to include".
+speech: noun; verb
+alt_words:
+  - 'blacklist (noun): blocklist, denylist, excludelist'
+  - 'whitelist (noun): allowlist, passlist, trustlist, safelist'
+  - 'blacklist (verb): block, deny, exclude'
+  - 'whitelist (verb): allow, trust, include'
```

Comment:
Here you can take advantage of some nestted terms here to keep this a bit neater:

You can see the [dude definition](https://www.selfdefined.app/definitions/dude/) as an example of how this might work.

Reply:
Oh this is awesome!